flink-runtime/src/main/java/org/apache/flink/runtime/operators/util/BitSet.java

```diff
@@ -56,8 +56,7 @@ public void setMemorySegment(MemorySegment memorySegment, int offset) {
 	 * @param index - position
 	 */
 	public void set(int index) {
-		Preconditions.checkArgument(index < bitLength && index >= 0, 
-			String.format("Input Index[%d] is larger than BitSet available size[%d].", index, bitLength));
+		Preconditions.checkArgument(index < bitLength && index >= 0);
```

Comment:
Can be changed to 

```
Preconditions.checkArgument(index < bitLength && index >= 0, 
  "Input Index[%d] is larger than BitSet available size[%d].", index, bitLength);
``` 
to perserve the error message.

Reply:
Just noticed your reply to the same comment on the JIRA issue. You are of course right, the varargs will create and fill an array which adds overhead.

So +1 to keep it as it is.